tools/sanity_checks.py

```diff
@@ -143,11 +145,26 @@ def test_releases(self):
                             self.check_source_url(name, wrap_section, ver)
                     if i == 0 and t not in self.tags:
                         with self.subTest(step='check_new_release'):
-                            self.check_new_release(name, info, wrap_section)
+                            self.check_new_release(name)
```

Comment:
In commit c8c74737cf6b82f6b200acc2d27b8d021ec5161c this stopped being used, indeed.

Can this cleanup be done in a separate commit for review purposes?

Reply:
That was indeed unrelated cleanup.